ffg/visitors/initialization_visitor.py

```diff
@@ -242,6 +320,18 @@ def visit_bit_vector_concatenation(self, operator: BitVectorConcatenation):
             operator.size = operator.operator_1.size + operator.operator_2.size
             self._is_var[operator] = self._is_var[operator.operator_1] and self._is_var[operator.operator_2]
 
+    def visit_bit_vector_ite(self, operator: BitVectorIte):
+        if operator in self._size:
+            operator.size = self._size[operator]
+            self._size[operator.operator_1] = operator.size
+            self._size[operator.operator_2] = operator.size
+            self._visit_ite(operator, BITVECTOR_OPERATOR)
+        else:
+            self._visit_ite(operator, BITVECTOR_OPERATOR)
+            operator.size = max(
+                [operator.operator_1.size, operator.operator_2.size])
+            self._is_var[operator] = self._is_var[operator.operator_1] and self._is_var[operator.operator_2]
```

Comment:
Not sure here, I am not sure whether we want to enforce that the size of both branches is the same or not, wdyt?